packages/arb-token-bridge-ui/src/components/TransferPanel/TokenDepositCheckDialog.tsx

```diff
@@ -110,8 +57,35 @@ export function TokenDepositCheckDialog(props: TokenDepositCheckDialogProps) {
   }, [type, symbol])
 
   return (
-    <Dialog {...props} title={title}>
-      <p className="pt-4">{textContent}</p>
+    <Dialog {...props} title={title} className="flex flex-col gap-4">
+      <p>{textContent}</p>
```

Comment:
Not using `TokenInfo` component at all because the token can be a native token which doesn't have an address. It's cleaner to keep it as it was, which means without the token image/name/address.